AmericanMedicalAssociation/AMA-style-guide
# Ama 28 hero cta link atom

https://palantir.atlassian.net/browse/AMA-28

Adds the Links directory under atoms, adds the Read More link atom. Wraps the Hero CTA in a link so that the entire molecule is clickable.

Steps to test:

1. Navigate to Atoms > Links > Link Read More
2. Observe that Read more is an atom.
3. Navigate to Molecules > Blocks > Hero CTA.
4. Observe that the Hero CTA molecule displays the Read More link.
5. Observe that upon hovering on the Hero CTA, the background of the text changes to #e6e6e6 and Read More is underlined.
